src/components/ux-reveal/ux-reveal.scss

```diff
@@ -6,6 +6,8 @@
 	}
 
 	.reveal-modal {
+		position: absolute;
```

Comment:
By isolating it from the main container, we are able to listen to the click on the background keeping the actual content modal out of the context. When the content is within the background container then any event listener on the content will bubble up to the background container. Isolating it will prevent this issue.